.circleci/config.yml

```diff
@@ -110,15 +107,18 @@ workflows:
           name: "Ensure artifact isolation, inform of boxed math"
           jdk_version: openjdk8
           lein_test_command: lein with-profile -dev,+check check
-          context: JFrog
+          context:
+            - Github
```

Comment:
this context isn't required for these steps, only for the ncrw-verify one. right?